### PR TITLE
NEWS: add v3.0.5

### PR DESCRIPTION
**Pull request by gyuho**
/cc @heyitsanthony @xiang90

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,11 @@
+etcd v3.0.5 (2016-08-19)
+
+  - v3 etcdctl get command now correctly returns results with --limit flag
```
Comment (xiang90) [inline NEWS]: we do not need to talk about bug fixes here i guess.

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+  - If etcd is using TLS without custom certificate authority, the discovery
```
Comment (xiang90) [inline NEWS]: @heyitsanthony Better words to make this more clear?

File: NEWS (hunk 1 above)
Comment (gyuho) [inline NEWS]: Just removed. Thanks.

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,10 @@
+etcd v3.0.5 (2016-08-19)
+
+  - If etcd is using TLS without custom certificate authority, the discovery
```
Comment (heyitsanthony) [inline NEWS]: SRV records (e.g., infra1.example.com) must match the discovery domain (i.e., example.com) when using the default certificate authority.

**Comment by gyuho** [comment]
@heyitsanthony PTAL. Thanks.

**Comment by xiang90** [comment]
lgtm

**Pull request merged**